### DISCUSSION TITLE
Sun setting sling box

### DISCUSSION THREAD

**Discussion by Gerry2591** [GerryDazoo/Slinger#54] [General]
Thank you Gerry,  I have setup the server on a windows server it works flawlessly up to now.
Question:  when dish network closes down the server is their a possibility that they would reset all of our sling boxes.  If so is there still a possibility to recover the finderid and passwords.  Can we protect from that eventuality.

thanks

**Comment by GerryDazoo** [comment]
We can live without the finderids but not the passwords. 
I'm afraid I can't read Mr. Sling's mind but I think it's very unlikely. 
If you're especially paranoid, you could make sure there are no open port maps on your router that would allow outside access to your slingbox.

**Comment by theweatherguy** [comment]
Interestingly, this is what I found on Google tonight:

_On behalf of the Sling Media team, we'd like to remind you that Slingbox servers will be permanently taken offline November 9, 2022, at which point all Slingbox devices and services will become inoperable._

Notice they say _**devices**_, not just the service.  I’m going to take Gerry’s advice for the paranoid and  close my port forwarding on my router to my slingboxes and switch to slinger a few days in advance.  I just hope they didn’t pre-program the boxes to become doorstops on a time-basis in advance.

**Reply by sp222222** [reply]
I'm going to unplug my slingboxes for 48hours before and after November 9th. I'm taking NO chances. This is one of the reasons I wish my other 500 I got 3 weeks ago from ebay that I cannot change the default port from 5201. I hope it makes the post 11/11/2022 date

**Reply by theweatherguy** [reply]
Gerry's Slinger is working so good, I've already removed the port forwarding to 5201 from my router on both my and my wife's slingboxes (both are 350s)  and we're using the new setup.  I could be wrong, but I don't think the boxes need to be unplugged on Nov 9th.  I think you just need to disable the port forwarding on your router before the deadline. (If I'm wrong about this, kindly correct me.)   Strangely, I think both of my boxes seemed to use 5201 as their port, but of course, they had different IP addresses dynamically assigned by my router that I changed to static addresses.

**Reply by sp222222** [reply]
do you use both from external network at the same time? I didn't think more than one port could be used at same time on same network. idk, i'm not cisco certified :)

**Reply by theweatherguy** [reply]
yes.  I will add the instructions for two slingboxes to the  bottom of the Mac Step-by-Step Tutorial Discussion

**Reply by theweatherguy** [reply]
I should add that I have a WireGuard VPN server running on my home network. I access my internal home network through that and stream connected to that outside the home.   I don't use an open port or port forwarding to access slinger.

**Comment by tom76017** [comment]
This is from the Singbox.com front page.  I unplugged my modem and everything on my LAN worked fine.

> 9/9/2022
> On behalf of the Sling Media team, we’d like to remind you that Slingbox servers will be permanently taken offline November 9, 2022, at which point all Slingbox devices and services will become inoperable.
